src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from 'src/app/services/user-service/user.service';
+import { User } from 'src/app/models/user';
+import { Router } from '@angular/router';
+import { environment } from '../../../environments/environment';
+import { HttpClient } from '@angular/common/http';
+import { AuthService } from 'src/app/services/auth-service/auth.service';
+
+@Component({
+	selector: 'app-login',
+	templateUrl: './login.component.html',
+	styleUrls: ['./login.component.css']
+})
+
+/**
+ * This is the login component
+ */
+export class LoginComponent implements OnInit {
+
+	users: User[] = [];
+	allUsers: User[] = [];
+
+	chosenUser: User;
+	chosenUserFullName: string = '';
+	userName: string = '';
```

Comment:
Why have these additional fields? Aren't they accessible through the "chosenUser" field?

Reply:
-- used for display/search --